### PR TITLE
init index and init stylesheet

### PR DESCRIPTION
**Pull request by willbenmitch**
Initialize an index.html

Initialize a styles.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html>
+    <head>
+        <title>color.js</title>
```
Comment (justinbansal) [inline index.html]: Please revise the title.

Comment (justinbansal) [inline index.html]: Please revise the title.

Comment (justinbansal) [inline index.html]: Please revise the title.

Comment (justinbansal) [inline index.html]: Please revise title.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <link rel="stylesheet" type="text/css" href="css/styles.css">
+<!--javascript-->
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+    </head>
+    <body>
+        <h1>Employee Names!</h1>
+        <ul>
+            <li><a href="employees/johnny_football.html">Johnny Football</a></li>
+            <li><a href="employees/cathy_wicket.html">Cathy Wicket</a></li>
+            <li><a href="employees/davey_chase.html">Davey Chase</a></li>
```
Comment (justinbansal) [inline index.html]: Please add closing tag for ul

**Comment by willbenmitch** [comment]
@justinbansal changes made! How's it look?

**Pull request merged**